src/nunjucks/moduk/components/cookie-banner/macro.njk

```diff
@@ -0,0 +1,5 @@
+{% from "govuk/components/cookie-banner/macro.njk" import govukCookieBanner %}
+
+{% macro modukCookieBanner(params) %}
+    {{ govukCookieBanner(params) }}
```

Comment:
Would be good to stick to 2-space indentation like elsewhere here:

```suggestion
  {{ govukCookieBanner(params) }}
```

Reply:
will add it to editorconfig